packages/theming/apple-theme/src/appleColors.types.macos.ts

```diff
@@ -215,6 +215,7 @@ interface FluentAppleButtonTokens {
   neutralBackgroundInverted: ColorValue;
   neutralForegroundInverted: ColorValue;
   neutralForeground2: ColorValue;
+  neutralBackground1: ColorValue;
   neutralBackground2: ColorValue;
```

Comment:
Outside of this review, but if we do get a finalized design, I wonder if we need to update the FUA macOS button too.. probably for the FUA crew to do.